### PR TITLE
Add employees page

### PR DESCRIPTION
**Pull request by renzo-x**
## Description

This PR includes the employees page and the employee view page for the system. I've included the following features: 

- Creating, viewing, updating, and deleting employees

- Assigning benefits, earnings, and deductions to an employee

## Type of change
New feature (adds new functionality)

### CODE REVIEW

**Comment by torodeveloper** [comment]
In this PR #2 by Kyle, he created a read-only details view for payslips similar to your employee details. I think we can create a reusable component for that portion of UI. Can one of you work on that?

**Comment by torodeveloper** [comment]
image

A newly added employee has an empty name.

**Comment by renzo-x** [comment]
> In this PR #2 by Kyle, he created a read-only details view for payslips similar to your employee details. I think we can create a reusable component for that portion of UI. Can one of you work on that?

**Comment by renzo-x** [comment]
> In this PR #2 by Kyle, he created a read-only details view for payslips similar to your employee details. I think we can create a reusable component for that portion of UI. Can one of you work on that?

Yes, we'll be working on it. We decided to use the component that I created.

**Comment by renzo-x** [comment]
> A newly added employee has an empty name.

Fixed the missing name when adding an employee.
fix-add-employee

**Comment by torodeveloper** [comment]
Can you rebase this to the latest master branch and resolve the conflicts?

File: components/navigation-bar/stylesheets/index.css
Comment (torodeveloper) [inline components/navigation-bar/stylesheets/index.css:1]: Do we still need this?

File: stylesheets/employees.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+.primary-btn > .default {
+    color: #f9f9f9;
```
Comment (torodeveloper) [inline stylesheets/employees.css]: Is that the same colors we have in the app theme?

**Review by torodeveloper — CHANGES_REQUESTED** [review]

File: stylesheets/employees.css (hunk 1 above)
Comment (renzo-x) [reply stylesheets/employees.css]: no sir

File: components/navigation-bar/stylesheets/index.css
Comment (renzo-x) [reply components/navigation-bar/stylesheets/index.css:1]: No. Alaina included another css styles for it.

File: stylesheets/employees.css (hunk 1 above)
Comment (torodeveloper) [reply stylesheets/employees.css]: We already have these [button colors in the app theme](https://github.com/torodeveloper/payroll-system-bellini-app/pull/1#issuecomment-2068368939). Can we use it instead?

Comment (renzo-x) [reply stylesheets/employees.css]: Done, sir. Is it okay if I used the #5 PR to create the changes?

[The commit](https://github.com/torodeveloper/payroll-system-bellini-app/pull/5/commits/1645e8b49543621fb21fe6b970cb3f73a3afeedb)

Comment (torodeveloper) [reply stylesheets/employees.css]: No, sorry we need that in this PR. We have to merge this PR first.

Comment (renzo-x) [reply stylesheets/employees.css]: Done with the update.

File: components/navigation-bar/stylesheets/navigation-bar.css
Comment (torodeveloper) [inline components/navigation-bar/stylesheets/navigation-bar.css:1]: Is this change intended?

Comment (renzo-x) [reply components/navigation-bar/stylesheets/navigation-bar.css:1]: yes sir

Comment (renzo-x) [reply components/navigation-bar/stylesheets/navigation-bar.css:1]: Scratch that. I removed it. I'll just use the default style.

**Comment by torodeveloper** [comment]
Can you rebase it again to the latest master?

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "payroll-system",
   "type": "app",
-  "defaultPage": "home",
+  "defaultPage": "employees",
```
Comment (torodeveloper) [inline app.json]: Can you revert this? We already set the payroll page as the default page.

Comment (renzo-x) [reply app.json]: yep, noted sir.

**Comment by renzo-x** [comment]
> Can you rebase it again to the latest master?

Done rebasing.

**Comment by torodeveloper** [comment]
You still have irrelevant file changes. For example the benefits page and its modals. We should not see those on your changes.

**Comment by renzo-x** [comment]
> You still have irrelevant file changes. For example the benefits page and its modals. We should not see those on your changes.

sir, are you referring to the benefits for an employee? That was intended since we like to assign benefits to an employee in the employee-view page.

Also, there was an error, the drop down menu to show the  tables for benefits, eranings, and deductions is missing. I'm working on a fix.

image

**Comment by renzo-x** [comment]
> You still have irrelevant file changes. For example the benefits page and its modals. We should not see those on your changes.

Oh, about the benefits page of Alaina. I'll look into it.

**Review by torodeveloper — APPROVED** [review]
